scene/3d/cpu_particles_3d.cpp

```diff
@@ -698,12 +726,12 @@ void CPUParticles3D::_particles_process(double p_delta) {
 		double restart_phase = double(i) / double(pcount);
 
 		if (randomness_ratio > 0.0) {
-			uint32_t seed = cycle;
+			uint32_t _seed = cycle;
 			if (restart_phase >= system_phase) {
-				seed -= uint32_t(1);
+				_seed -= uint32_t(1);
 			}
-			seed *= uint32_t(pcount);
-			seed += uint32_t(i);
+			_seed *= uint32_t(pcount);
+			_seed += uint32_t(i);
 			double random = double(idhash(seed) % uint32_t(65536)) / 65536.0;
```

Comment:
```suggestion
			double random = double(idhash(_seed) % uint32_t(65536)) / 65536.0;
```
Assuming, as the `_seed` is unused